fastlane/actions/increment_version_code.rb

```diff
@@ -0,0 +1,50 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      INCREMENTED_VERSION_CODE = :INCREMENTED_VERSION_CODE
+    end
+
+    class IncrementVersionCodeAction < Action
```

Comment:
I mentioned in a previous comment to populate the build number from Appcenter, if that is possible. Then we don't need an autoincrement since it'll just pull from appcenter

Reply:
This action is still kinda needed, because this writes the actual value to the properties file, unless there's a more direct way to do so without a find and replace. I tried to specify a value directly through the Gradle fastlane action like so:

`
properties: {
        "archivesBaseName" => "sdkdemo_#{versionNumber}",
        "versionCode" => "#{someValue}"
}
`
But it didn't appear to override the value already set. Can play around with it a bit more, if I can directly manipulate the properties without fussing with regex I'd be happy.